p2p/simulations/network_test.go

```diff
@@ -140,6 +214,29 @@ func TestNetworkSimulation(t *testing.T) {
 	}
 }
 
+type placeboService struct {
```

Comment:
Is it helpful to set this `placeboService` into some helper module or something? There are now at least three other tests using some kind of `noopService` (`node/utils_test.go`, ` swarm/network/simulation/simulation_test.go` and this one)

Reply:
yeah we should do that!

Reply:
done and a good point. i used the noopService in `node/`